### PR TITLE
[KOGITO-836] - Provide a way to automatically increase the Kogito ver…

### PR DESCRIPTION
**Pull request by spolti**
…sion on the image and modules

Signed-off-by: spolti <fspolti@redhat.com>

### CODE REVIEW

**Review by Kaitou786 — APPROVED** [review]

**Review by radtriste — COMMENTED** [review]
Please add a README entry.
(and also `pip install ruamel.yaml` needed to execute the new python script)

**Comment by radtriste** [comment]
also getting
```
$ python scripts/manage-kogito-version.py --bump-to 0.99.0
Provided version 0.99.0 does not match the expected regex - \d.\d.\d$|\d.\d.\d-rc\d
```

**Review by ricardozanini — APPROVED** [review]

**Review by Kaitou786 — APPROVED** [review]

**Review by sutaakar — APPROVED** [review]

**Comment by spolti** [comment]
sure thing, will do.
about the regex, I forget that minor version will be greater than 10 in a few weeks LOL

will fix it as well.

**Comment by spolti** [comment]
@radtriste can you please take a look again?

File: scripts/README.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+$ python manage-kogito-version.py --bump-to 1.0.0  
+```
+
+The command above will update all the needed files to the version 1.0.0. These changes includes updates on
+
+ - all cekit modules
+ - image.yaml file descriptor
+ - kogito-imagestream.yaml
+ 
+#### Script dependencies
```
Comment (radtriste) [inline scripts/README.md]: I would put this one before the execution of the script, as simple requirements

File: scripts/README.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+
+Staging images are the release candidates which are pushed mainly after big changes that has direct impact on how
+the images will behave and also when new functionality is added.
+
+This script is called as the last step of the `make push-staging` command defined on the [Makefile](../Makefile).
+
+It will look for the current RC images available on [quay.io](https://quay.io/organization/kiegroup) to increase the rc tag 
+accordingly then push the new tag so it can be tested by others. 
+
+#### Script dependencies
```
Comment (radtriste) [inline scripts/README.md:42]: also here, would put the requirements before

**Review by radtriste — APPROVED** [review]
ok for me

two-ish minor comments

**Review by radtriste — APPROVED** [review]
lgtm

**Pull request merged**